devito/stencilkernel.py

```diff
@@ -97,7 +98,8 @@ def __init__(self, stencils, **kwargs):
 
         # Introduce C-level profiling infrastructure
         self.sections = OrderedDict()
-        nodes = self._profile_sections(nodes)
+        if self.profiler:
```

Comment:
Yes, but the point is that *if* one PR introduces a problem, then it should also fix it -- it shouldn't be the case that one PR introduces a problem which *someone* maybe/one day/in the future/perhaps/... will fix

This is an approach I was trying to avoid.